cumulusci/core/config/__init__.py

```diff
@@ -6,22 +6,61 @@
 from cumulusci.core.config.BaseConfig import BaseConfig
 
 # inherit from BaseConfig
-from cumulusci.core.config.BaseTaskFlowConfig import BaseTaskFlowConfig
-from cumulusci.core.config.ConnectedAppOAuthConfig import ConnectedAppOAuthConfig
-from cumulusci.core.config.FlowConfig import FlowConfig
+
+from cumulusci.core.config.MergedConfig import MergedConfig
+from cumulusci.core.config.MergedConfig import MergedYamlConfig
+
+
+class ConnectedAppOAuthConfig(BaseConfig):
+    """ Salesforce Connected App OAuth configuration """
+
+    pass
+
+
+class FlowConfig(BaseConfig):
+    """ A flow with its configuration merged """
+
+    pass
+
+
 from cumulusci.core.config.OrgConfig import OrgConfig
-from cumulusci.core.config.ServiceConfig import ServiceConfig
-from cumulusci.core.config.TaskConfig import TaskConfig
+
+
+class ServiceConfig(BaseConfig):
+    pass
+
+
+class TaskConfig(BaseConfig):
+    """ A task with its configuration merged """
+
+    pass
+
+
+from cumulusci.core.config.BaseTaskFlowConfig import BaseTaskFlowConfig
+
 
 # inherit from BaseTaskFlowConfig
-from cumulusci.core.config.BaseGlobalConfig import BaseGlobalConfig
 from cumulusci.core.config.BaseProjectConfig import BaseProjectConfig
 
 # inherit from OrgConfig
 from cumulusci.core.config.ScratchOrgConfig import ScratchOrgConfig
 
-# inherit from BaseGlobalConfig
-from cumulusci.core.config.YamlGlobalConfig import YamlGlobalConfig
-
 # inherit from BaseProjectConfig
-from cumulusci.core.config.YamlProjectConfig import YamlProjectConfig
+from cumulusci.core.config.BaseGlobalConfig import BaseGlobalConfig
+
+
+__all__ = [
+    "FAILED_TO_CREATE_SCRATCH_ORG",
+    "BaseConfig",
+    "MergedConfig",
+    "MergedYamlConfig",
+    "ConnectedAppOAuthConfig",
+    "FlowConfig",
+    "OrgConfig",
+    "TaskConfig",
+    "ServiceConfig",
+    "BaseTaskFlowConfig",
+    "BaseGlobalConfig",
+    "BaseProjectConfig",
+    "ScratchOrgConfig",
+]
```

Comment:
Is there a problem you're solving by listing these explicitly?

Reply:
I thought there was but it turned out to be irrelevant.